compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
```

Comment:
for now should not be commited

Reply:
ok, but why it does not work in auto mode while starting spring boot app?